presto-main/src/main/java/com/facebook/presto/cost/CostCalculatorUsingExchanges.java

```diff
@@ -169,7 +193,9 @@ public PlanNodeCostEstimate visitExchange(ExchangeNode node, Void context)
         @Override
         public PlanNodeCostEstimate visitTableScan(TableScanNode node, Void context)
         {
-            return cpuCost(getStats(node).getOutputSizeInBytes()); // TODO: add network cost, based on input size in bytes?
+            // memory to cpu througput is about 10GB/s
+            // disk to memory is 100MB/s - 100 slower
+            return cpuCost(getStats(node).getOutputSizeInBytes() * 100); // TODO: add network cost, based on input size in bytes?
```

Comment:
This assumes you read from disk, which may or may not be the case. Cost of reading is something a connector should be able to provide (consider adding relevant TODO in the code). 

Even if you assume we read from HDFS, this is both CPU cost and network cost.

What about assuming this is `(cpu=dataSize, mem=0, net=dataSize)` for now?